app/controllers/api/v1x2/order_processes_controller.rb

```diff
@@ -0,0 +1,37 @@
+module Api
+  module V1x2
+    class OrderProcessesController < ApplicationController
+      include Api::V1x1::Mixins::IndexMixin
+      include Api::V1x0::Mixins::ShowMixin
+
+      def index
+        collection(OrderProcess.all)
+      end
+
+      def create
+        order_process = authorize(OrderProcess.new(writeable_params_for_create))
```

Comment:
There is no need for `authorize` on create? Haven't looked at this code in a while but that stuck out.

Reply:
@syncrou We need to check `authorize` on create. According to the permissions, only catalog `admin` is allow to create `OrderProcess`